### DISCUSSION TITLE
求助滚轮快切功能怎么用啊，没有找到设置

### DISCUSSION THREAD

**Discussion by SpaceThing** [shuax/MouseInc.Settings#292] [Q&A]
![image](https://user-images.githubusercontent.com/61451659/173543269-881329e4-f00c-4bed-b132-122741ac247f.png)

**Comment by enximi** [comment]
![image](https://user-images.githubusercontent.com/70036307/174489913-bc42cf2c-be2f-47cf-a4ab-451787b381bd.png)

**Reply by uiop321** [reply]
这个功能没有实现呢，不知道哪里出了问题。
